setup.py

```diff
@@ -79,6 +79,7 @@
         'bitcoin==1.1.42',
         'flask==0.10.1',
         'requests==2.9',
+        'gunicorn',
```

Comment:
this can help: http://python-packaging-user-guide.readthedocs.org/en/latest/requirements/#install-requires

> It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

it appears that inequality operators (`<`, `<=`, `>`, `>=`) are preferred